packages/accounts-app/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
   "compilerOptions": {
-    "jsx": "react",
```

Comment:
question: why is this removed?

Reply:
I put it in the root one, since it's shared by everyone